### PR TITLE
Adds pagination to `pullrequests` routes: `/stale` and `/open`

### LINKED ISSUE #125: Add `page` and `n` query params for `pullrequests` `/open/` and `/stale` APIs
Currently we have `page` query param for `/pullrequests/open` route, but we don't have `n` (number of pull requests displayed on one page) for this route and we're missing both the query params for `/pullrequests/stale` route

Easy one. Please comment down if you want to work on this issue.

### PR DESCRIPTION
**Pull request by swarajpure**
Fixes #125

### CODE REVIEW

File: controllers/pullRequests.js [hunk 1]
```diff
@@ -37,7 +37,8 @@ const getUserPRs = async (req, res) => {
  */
 const getStalePRs = async (req, res) => {
   try {
-    const { data } = await githubService.fetchStalePRs()
+    const { size, page } = req.query
```
Comment (ankurnarkhede) [inline controllers/pullRequests.js:40]: Can you add these query params in the schema for the routes?

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]
- Query params needed to be added in the schema for the routes.
- Need unit and integration tests.

File: controllers/pullRequests.js (hunk 1 above)
Comment (swarajpure) [reply controllers/pullRequests.js:40]: Sure, will do that.

**Comment by swarajpure** [comment]
>     * Query params needed to be added in the schema for the routes.
> 
>     * Need unit and integration tests.

Integration tests in PR #326

File: controllers/pullRequests.js (hunk 1 above)
Comment (swarajpure) [reply controllers/pullRequests.js:40]: Done. Please check now

**Comment by swarajpure** [comment]
Unit tests will be a part of PR #326

File: controllers/pullRequests.js (hunk 1 above)
Comment (swarajpure) [reply controllers/pullRequests.js:40]: image

**Review by ankurnarkhede — APPROVED** [review]

**Pull request merged**